lua/weapons/gmod_tool/stools/starfall_processor.lua

```diff
@@ -200,7 +209,7 @@ function TOOL:Think()
 	-- Ghost code
 	if (SERVER and game.SinglePlayer()) or (CLIENT and not game.SinglePlayer()) then
 		local model = self:GetClientInfo("ScriptModel")
-		if model=="" then
+		if model=="" or not (util.IsValidModel(model) and util.IsValidProp(model)) then
```

Comment:
Could you fix this one too?

Reply:
Do you mean revert it to the original?

Reply:
No, it should match the other one shouldn't it?